xbmc/threads/test/TestThreadLocal.cpp

```diff
@@ -35,7 +33,7 @@ class Thingy
 
 Thingy* staticThingy = NULL;
 CEvent gate;
-ThreadLocal<Thingy> staticThreadLocal;
+thread_local Thingy* staticThreadLocal;
```

Comment:
What's the point of this test if we use `thread_local` anyway?

Reply:
It also tests Event

Reply:
There is already `TestEvent.cpp` - this file was specifically for `ThreadLocal`

Reply:
I know. I wrote it about 6 years ago :-) It uses CEvent also so I left it. I can trim it down.

Reply:
Is there some specific interaction between `CEvent` and `thread_local`? If not, moving the tests to `TestEvent` would make them clearer about their purpose.